source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -32,25 +33,31 @@ export default class PinCode extends Component<Props> {
   focusKey = 0;
   add = false;
 
-  onChange = (inputValue: ?number, key: number) => {
+  onChange = (inputValue: ?number | ?string, key: number) => {
     const { value, onChange } = this.props;
+
     const inputNewValue =
-      inputValue !== null && inputValue !== undefined
+      inputValue !== null && inputValue !== undefined && inputValue !== 'NaN'
```

Comment:
`inputValue !== 'NaN'` will always be true, unless its value is actually `NaN`. Please use `isNaN(inputValue)` instead